### PR TITLE
Ability to copy files from blank-app

### PR DESCRIPTION
**Pull request by hallelujah**
- works with local install aka `git clone`
- works with `luarocks make --local`

### CODE REVIEW

File: src/apicast-cli/cmd/generator.lua [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+  local destiny = opts.path or path.currentdir()
 
-  local raw_app = current_dir .. '/blank-app'
-  local new_app = destiny .. "/" .. name
-  assert(os.execute("cp -a '" .. raw_app .. "' '" .. new_app .. "'"))
+  local raw_app = path.join(utils_path.find(), 'apicast-cli/blank-app')
+  local new_app = path.join(destiny, name)
+  local command = {"cp", "-a", raw_app, new_app}
+  print('Generating new project into ' .. new_app)
+  assert(os.execute(table.concat(command, " ")))
```
Comment (macejmic) [inline src/apicast-cli/cmd/generator.lua:15]: 🍤

Comment (mikz) [reply src/apicast-cli/cmd/generator.lua:15]: it should also escape parameters 🙂 otherwise it breaks on spaces, `$`, `\` etc

Comment (hallelujah) [reply src/apicast-cli/cmd/generator.lua:15]: Agree! Will improve later.
@mikz we may need to extract this kind of things to the utils from apicast (if they exist of course)

Comment (mikz) [reply src/apicast-cli/cmd/generator.lua:15]: There is something. https://github.com/3scale/apicast/compare/luarock

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+LUA_PATH="$PWD/src/?.lua;$LUA_PATH"
```
Comment (macejmic) [inline .env:1]: shouldn't be there, noupe?

i've added `.env` to `.gitignore`

File: src/apicast-cli/utils/path.lua [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  end
+end
+
+local find_dev_directory = function()
+  -- If we run the script apicast-cli
+  -- Look for the stacktrace and the file where the function was called
+  local package_path = (debug.getinfo(1).source):match("^@?(.-)/apicast%-cli/utils/path%.lua$")
+  local f = package_path and io.open(path.join(package_path, "apicast-cli/utils/path.lua"), 'r')
+
+  if f then
```
Comment (macejmic) [inline src/apicast-cli/utils/path.lua:23]: we had here some `else statement` before, didn't we?

File: .env (hunk 2 above)
Comment (hallelujah) [reply .env:1]: I forced it :)
It is good to have it but feel free to remove if you do not like it

File: src/apicast-cli/utils/path.lua (hunk 3 above)
Comment (hallelujah) [reply src/apicast-cli/utils/path.lua:23]: Removed because there is no else encountered yet.
But I agree we need to handle the `nil` value

**Pull request merged**